packages/near-membrane-base/src/membrane.ts

```diff
@@ -611,6 +644,34 @@ export function createMembraneMarshall() {
             return fn;
         }
 
+        function isLiveObject(value: any): boolean {
+            if (value === null || value === undefined || value === ObjectProto) {
```

Comment:
```suggestion
            if (value === ObjectProto) {
```

the value should not be type any, it should be a proxy target type, this should never be called with a thing that is not a valid proxy target.

Reply:
Fixed